simulation/navigator_2Dsim/launch/owl.launch

```diff
@@ -0,0 +1,13 @@
+<launch>
+
+	<arg name="gui" default="False"/>
+	<param name="inertia" value="[2000, 2000, 4000]" />
```

Comment:
Should just have a mass and rotational inertia value, as it can't have a different x and y mass. Also it should be documented what units everything is in

Reply:
Units are in the main python file but it does make more sense to put them in the launch file where people will actually be changing them.

Reply:
See first part of comment